validator/api/src/main/java/tech/pegasys/teku/validator/api/KeyStoreFilesLocator.java

```diff
@@ -55,11 +55,13 @@ public void parse() {
       }
 
       final List<String> entry = Splitter.on(pathSeparator).limit(2).splitToList(currentEntry);
-      parseEntry(entry.get(0), entry.get(1));
+      pathMap = parseEntry(entry.get(0), entry.get(1));
```

Comment:
I think we should just pass `pathMap` in as an argument and then `parseEntry` would add to that map. Otherwise each entry we parse replaces all the previous entries.